src/screens/UserPortal/Campaigns/Campaigns.module.css

```
@@ -29,17 +29,144 @@
+  box-shadow: none;
+}
+
+.addPledgeButton:disabled {
+  background-color: var(--color-gray-200);
+  color: var(--color-gray-500);
+  opacity: 1;
+}
+
+.progressPie {
+  --pie-fill-color: var(--color-blue-500);
+  width: var(--space-8);
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)